backend/db/manticore_db.py

```diff
@@ -779,11 +792,46 @@ async def prepare_payload_doc(
         return docid, payload
 
     async def delete(self, dto: EntityDTO) -> bool:
+        id = self.generate_doc_id(dto)
+        command = f"DELETE FROM key_value_pairs WHERE key = '{id}');"
+        try:
+            self.utilsApi.sql(command)
+        except Exception as e:
+            logger.error(f"Error at ManticoreDB.delete: {e.args}")
+            return False 
         return True
 
+    async def delete_keys(self, keys: list[str]) -> bool: # return error !
+        for key in range(list):
+            try:
+                sql_str = "delete from key_value_pairs where key = '{key}'" 
+                self.utilsApi.sql(sql_str)
+            except:
+                return False
+        return True
+ 
+
     async def delete_doc_by_id(self, id: str) -> bool:
```

Comment:
this function deletes the record from the corresponding table, use get_index_name_from_doc_id function to get the record index, and delete it from there